tasks/variable-setup.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Set datadog_checks variable
+  set_fact:
+    agent_datadog_checks: "{{ datadog_checks | default({}, true) }}"
```

Comment:
We do basically the same in `sanitize-checks.yml`, don't we? I think we can just do it there and not here.